core/trino-main/src/test/resources/hydra/README.md

```diff
@@ -0,0 +1,8 @@
+Certificate generation:
```

Comment:
See `core/trino-main/src/test/resources/cert/`. But consider comment below to decide if we need this at all.